packages/components/package.json

```diff
@@ -68,7 +68,8 @@
     "react-select": "1.0.0-rc.10",
     "react-sticky": "6.0.3",
     "reactn": "2.2.4",
-    "vis": "4.21.0"
+    "vis": "4.21.0",
+    "yarn": "^1.22.0"
```

Comment:
Was there a specific reason for needing to add yarn as a dependency here? I don't see any code changes in this PR that would warrant it.

Reply:
no, that wasn't intentional, also not sure how the entry got into the yarn.lock file